src/functionalTest/java/uk/gov/hmcts/reform/bulkscan/orchestrator/AttachExceptionRecordWithOcrToExistingCaseTest.java

```diff
@@ -0,0 +1,120 @@
+package uk.gov.hmcts.reform.bulkscan.orchestrator;
+
+import io.restassured.RestAssured;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.MediaType;
+import org.springframework.test.context.ActiveProfiles;
+import uk.gov.hmcts.reform.bulkscan.orchestrator.controllers.CcdCallbackController;
+import uk.gov.hmcts.reform.bulkscan.orchestrator.dm.DocumentManagementUploadService;
+import uk.gov.hmcts.reform.bulkscan.orchestrator.helper.CaseSearcher;
+import uk.gov.hmcts.reform.bulkscan.orchestrator.helper.CcdCaseCreator;
+import uk.gov.hmcts.reform.bulkscan.orchestrator.helper.EnvelopeMessager;
+import uk.gov.hmcts.reform.bulkscan.orchestrator.services.ccd.CcdApi;
+import uk.gov.hmcts.reform.bulkscan.orchestrator.services.ccd.CcdAuthenticator;
+import uk.gov.hmcts.reform.bulkscan.orchestrator.services.ccd.CcdAuthenticatorFactory;
+import uk.gov.hmcts.reform.ccd.client.model.CallbackRequest;
+import uk.gov.hmcts.reform.ccd.client.model.CaseDetails;
+import uk.gov.hmcts.reform.logging.appinsights.SyntheticHeaders;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.UUID;
+import java.util.concurrent.TimeUnit;
+
+import static java.time.Instant.now;
+import static java.util.Collections.emptyList;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.awaitility.Awaitility.await;
+import static org.hamcrest.Matchers.empty;
+import static uk.gov.hmcts.reform.bulkscan.orchestrator.services.ccd.definition.ExceptionRecordFields.ATTACH_TO_CASE_REFERENCE;
+
+@SpringBootTest
+@ActiveProfiles("nosb")
+class AttachExceptionRecordWithOcrToExistingCaseTest {
+
+    @Value("${test-url}")
+    String testUrl;
+
+    @Value("${document_management.url}")
+    String documentManagementUrl;
+
+    @Value("${document_management.context-path}")
+    String dmContextPath;
+
+    @Autowired CcdApi ccdApi;
+    @Autowired CcdCaseCreator ccdCaseCreator;
+    @Autowired CaseSearcher caseSearcher;
+    @Autowired EnvelopeMessager envelopeMessager;
+    @Autowired DocumentManagementUploadService dmUploadService;
+    @Autowired CcdAuthenticatorFactory ccdAuthenticatorFactory;
+
+    @Disabled
+    @Test
+    void should_update_case_with_ocr() throws Exception {
```

Comment:
you will always have scanned docs when we update OCR so best to add docs and OCR country update ?

Reply:
Documents are irrelevant afaik.
Orchestrator just sends entire case to external service.

Reply:
(the sample test service only uses OCR and only address part of it